frontend/src/__tests__/cypress/cypress/pages/components/table.ts

```diff
@@ -17,7 +17,7 @@ export class TableRow extends Contextual<HTMLTableRowElement> {
   }
 
   findKebabAction(name: string): Cypress.Chainable<JQuery<HTMLElement>> {
-    return this.find().findKebabAction(name);
+    return this.find().findKebabAction(name).should('exist').and('be.visible');
```

Comment:
Can you check please if this is breaking the Mock Tests? 
https://github.com/opendatahub-io/odh-dashboard/actions/runs/12427145849/job/34696500427?pr=3580

Reply:
Fixed